protected/models/media/media.vo.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * ValuesObject Class For Example Model
+ */
+class ExampleVO extends ValuesObject
+{
+    /**
+     * Get Example ID
+     *
+     * @return int Example ID
+     */
+    public function getID(): int
```

Comment:
Codacy found an issue: [Public method name "ExampleVO::getID" is not in camel caps format](https://app.codacy.com/manual/common/buryak/pullRequest?prid=5800482)